tools/Dashboard/DevHome.Dashboard/Views/DashboardView.xaml.cs

```diff
@@ -195,8 +195,8 @@ private void UnsubscribeFromWidgets()
 
     private async Task<bool> ValidateDashboardState()
     {
-        // Ensure we're not running as admin. Display an error and don't allow using the Dashboard if we are.
-        if (ViewModel.IsRunningAsAdmin())
+        // Ensure we're not running elevated. Display an error and don't allow using the Dashboard if we are.
+        if (ViewModel.IsRunningElevated())

```

Comment:
>IsRunningElevated [](http://example.com/codeflow?start=22&length=17)

When this throws exception, do we need to catch it somewhere in the stack? Will it crash the process?

Reply:
similar to my comment below, this should pretty much never throw except in exceptional cases where we would want a crash dump to root cause the issue.